Sources/NIOSSL/SSLContext.swift

```diff
@@ -123,14 +123,20 @@ private func alpnCallback(ssl: OpaquePointer?,
 ///
 /// - Warning: Avoid creating `NIOSSLContext`s on any `EventLoop` because it does _blocking disk I/O_.
 public final class NIOSSLContext {
+    internal typealias CertificateChainVerificationCallback = () -> EventLoopFuture<Void>
```

Comment:
Can we add the word additional here?

```suggestion
    internal typealias AdditionalCertificateChainVerificationCallback = () -> EventLoopFuture<Void>
```

Reply:
Also, it's probably a good idea for us to try to pass the `Channel` in to this callback, so that users have access to both the correct `EventLoop` _and_ to a way to find out what the context of the callback is.